vault/blog.2021.2021-09-27-markdown-on-mobile.md

```diff
@@ -0,0 +1,97 @@
+
+I'm really interested in seeing where GitJournal goes. I couldn't find other mobile apps that covered both notes in Markdown and git-integration, without needing to juggle multiple apps. With my [Dendron vaults](https://wiki.dendron.so/notes/6682fca0-65ed-402c-8634-94cd51463cc4.html), I mostly take [scratch notes](https://wiki.dendron.so/notes/5c213aa6-e4ba-49e8-85c5-1bdcb33ce202.html#scratch-note) or edit already-existing notes when using my phone. For more extensive usage, I hop on the laptop.[^1]
+
+## Markor
+
+![Editor and markdown-preview screenshots, on Android, for Markor](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-markor-screenshots.png)
+
+- **Cost:** Free
+- **OS:** Android
+- **Open Source:** Yes ([Apache 2.0, w/ special note on commercial license](https://github.com/gsantner/markor/blob/master/LICENSE.txt))
+- **Git integration:** No
+
+[Markor](https://gsantner.net/project/markor.html) is without git integration, but can be combined with an app like **GitJournal** or [MGit (Android)](https://manichord.com/projects/mgit.html) for syncing to a source repo.
+
+I liked both the _editor_ and _preview_ rendering in Markor more than the other apps I tried. Syntax highlighting is a nice touch while editing, and the previews look cleaner than both iA Writer and GitJournal.
+
+Though, the cons here are that Markor only works on Android and needs another app to handle git integration. Outside of that, I can quickly see why thousands of ratings have led to nearly a 5/5 on the Google Play.
+
+## iA Writer
+
+![Editor and markdown-preview screenshots, on Android, for iA Writer](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-iawriter-screenshots.png)
+
+- **Cost:** [$29.99](https://ia.net/downloads#apps)
+- **OS:** Android, iOS / iPad / Mac, Windows
+- **Open Source:** No
+- **Git integration:** No
+
+[iA Writer](https://ia.net/writer) was initially released 10 years ago, and has since evolved to work on multiple platforms. Though it doesn't have git integration, other apps like **GitJournal** or [MGit (Android)](https://manichord.com/projects/mgit.html) can be used for that.
+
+Want to feel like a designer, in a sleek interface with minimalist and Apple-like branding? This is made for you. The editor can work across all devices: phones, tablets, and desktops.
+
+Being a feature-full piece of software, including:
+
+- Style-checking to help improve note quality
+- Can export to Word
+- A larger list of all-the-things can be found on their [main landing page](https://ia.net/writer).
+
+## Other apps of interest
+
+Here is a slew of other apps mentioned in the Dendron docs, and mentioned in the community discord, with links.
```

Comment:
There are many mobile note-taking solutions that work with Markdown. Below are some additional tools that [Dendrologists](TODO:link to entry in [[Concepts|dendron://dendron.dendron-site/dendron.concepts]] explaining what a dendrologists are) have used for markdown on the go